lib/rspec/expectations/expectation_target.rb

```diff
@@ -42,7 +42,7 @@ def self.for(value, block)
         elsif block
           raise ArgumentError, "You cannot pass both an argument and a block to `expect`."
         else
-          new(value)
+          ValueExpectationTarget.new(value)
```

Comment:
Do we now ever instantiate an `ExpectationTarget`? Could/should this class be merged into it?

Reply:
Oh wait it's because `BlockExpectationTarget` inherits from it too right?